.github/workflows/security-scan.yml

```
@@ -0,0 +1,33 @@
+            **/package-lock.json
+
+      - name: Run Vulnerability Scanner
+        if: steps.changed-files.outputs.any_changed == 'true'
+        uses: fylein/vulnerability-scan-action@master
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          is_submodule: true
+
+      - name: Skip Vulnerability Scanner
+        if: steps.changed-files.outputs.any_changed != 'true'
+        run: echo "No changes to dependency files, skipping vulnerability scan."
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)